sda-commons-server-prometheus-example/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-  compile project(':sda-commons-server-starter')
+  compile project(':sda-commons-server-starter-swagger')
```

Comment:
Maybe we should use the new bundle everywhere?

Reply:
yes, we should use the new bundle everywhere except the `dependency-check` & the `build.gradle` of the root-project. Hopefully I've found every occurrence. Thanks for your comments!